### PR TITLE
ana/create addresses for other tendermint networks

### PR DESCRIPTION
**Pull request by Bitcoinera**
This is a work in progress, although looks good for now

### CODE REVIEW

File: src/cosmos-keys.ts [hunk 1]
```diff
@@ -49,16 +49,21 @@ export function getSeed(randomBytesFunc: (size: number) => Buffer = randomBytes)
...
+  return getNewWalletFromSeed(mnemonic, networkPrefix)
 }
 
 // NOTE: this only works with a compressed public key (33 bytes)
-export function getCosmosAddress(publicKey: Buffer): string {
+export function getCosmosAddress(publicKey: Buffer, networkPrefix: string): string {
   const message = CryptoJS.enc.Hex.parse(publicKey.toString('hex'))
   const address = CryptoJS.RIPEMD160(CryptoJS.SHA256(message) as any).toString()
-  const cosmosAddress = bech32ify(address, `cosmos`)
+  if (networkPrefix === `regen`) {
```
Comment (faboweb) [inline src/cosmos-keys.ts]: let's not handle the network specific logic in here. `networkPrefix` can be `bech32Prefix`. The integrating application then can take car of defining where to get the prefix from.

File: src/cosmos-keys.ts [hunk 2]
```diff
@@ -49,16 +49,21 @@ export function getSeed(randomBytesFunc: (size: number) => Buffer = randomBytes)
...
+    networkPrefix = `xrn:`
+  } else if (networkPrefix === `livepeer`) {
```
Comment (faboweb) [inline src/cosmos-keys.ts]: Nice idea. But I would propose that the integrating application handles this case to not pollute this code base

Comment (Bitcoinera) [reply src/cosmos-keys.ts]: OK, will refactor it

Comment (Bitcoinera) [reply src/cosmos-keys.ts]: Cool! You were fast

**Pull request merged**